test/e2e/Makefile

```
@@ -240,13 +240,26 @@ common/create-vg:
 	$(SUDO) losetup -f $(BACKING_STORE)/backing_store$(ID)_7
 	$(SUDO) vgcreate -y node$(ID)-thin1 $$($(SUDO) losetup -j $(BACKING_STORE)/backing_store$(ID)_7 | cut -d: -f1)
 	$(SUDO) lvcreate -T -n pool0 -L 4G node$(ID)-thin1
+	# Create Volume Group for Volume Health Monitoring Failure Check
+	truncate --size=2G $(BACKING_STORE)/backing_store$(ID)_8
+	truncate --size=2G $(BACKING_STORE)/backing_store$(ID)_9
+	# Setup loop devices
+	$(SUDO) losetup -f $(BACKING_STORE)/backing_store$(ID)_8; \
+	$(SUDO) losetup -f $(BACKING_STORE)/backing_store$(ID)_9; \
+	echo test | $(SUDO) cryptsetup open --type plain \
+		$$($(SUDO) losetup -j $(BACKING_STORE)/backing_store$(ID)_8 | cut -d: -f1) e$(ID); \
```

The dm-crypt device name may be hard to understand.
(Sorry, but I don't remember why I used the name "eX" for the device name in [my script.](https://github.com/topolvm/topolvm/pull/916#issuecomment-2241896266)..)
```suggestion
		$$($(SUDO) losetup -j $(BACKING_STORE)/backing_store$(ID)_8 | cut -d: -f1) crypt-$(ID); \
```